ci/jenkins/jobs/macros.yaml

```
@@ -229,7 +229,10 @@
           # Clean up dangling images generated in previous builds. Recent ones must be excluded
           # because they might be being used in other builds running simultaneously.
           docker image prune -f --filter "until=1h" || true
-          VERSION="$JOB_NAME-$BUILD_NUMBER" make
+          for i in 1 2
```

```suggestion
          for i in `seq 2`
```
Please add a comment of the reason of retry.